projects/storefrontlib/layout/a11y/keyboard-focus/persist/persist-focus.service.ts

```diff
@@ -18,15 +18,15 @@ export class PersistFocusService extends BaseFocusService {
   // We must bring this to a singleton map.
   protected focus = new Map<string, string>();
 
-  get(group?: string): string {
+  get(group?: string | null): string | undefined {
     return this.focus.get(group || GLOBAL_GROUP);
   }
 
   /**
    * Persist the keyboard focus state for the given key. The focus is stored globally
    * or for the given group.
    */
-  set(key: string, group?: string) {
+  set(key?: string, group?: string | null) {
```

Comment:
instead of making the `key` optional, let's still make it mandatory.

But make the type `string | undefined`. I don't think it makes sense just to allow customers to do .set()